docs/source/benchmarking/materials_science_and_engineering/systemcapabilities.rst

```diff
@@ -1,11 +1,11 @@
 System capabilities, operating ranges, and survival ranges ontology (SystemCapabilities)
```

Comment:
move the wording into the context, but here convert `System capabilities, operating ranges, and survival ranges ontology (SystemCapabilities)` to `System Capabilities Ontology (SystemCapabilities)`